app/src/main/java/com/example/marvelapp/api/util/Extensions.kt

```diff
@@ -22,17 +24,23 @@ fun CharacterEntity.transformToLocalCharacter(): Character {
     return Character(
         this.id,
         this.name,
-        this.description
+        this.description,
+        this.imageURL
     )
 }
 
 fun Character.transformToCharacterEntity(): CharacterEntity {
     return CharacterEntity(
         this.id,
         this.name,
-        this.description
+        this.description,
+        this.imageURL
     )
 }
 
+fun ImageResponse.getPathAndExtension(): String {
+    return this.path + "." + this.extension
```

Comment:
please use a more kotlin style way to concatenate

"${this.path}$DOT${this.extension}"

Reply:
done